qmail-rspawn.c

```diff
@@ -84,7 +95,9 @@ char *s; char *r; int at;
  int f;
  char *(args[5]);
 
- args[0] = "qmail-remote";
+ setup_qrargs();
```

Comment:
This repeats the same steps every time. Isn't there some way we could hook this into main()?

Reply:
Good idea.